src/reducers/resources.js

```diff
@@ -512,14 +512,14 @@ const addFirstValue = (state, propertyKey) => {
     case "InputLiteral":
       return addValueToNewState(
         state,
-        newBlankLiteralValue(property),
+        newBlankLiteralValue(property, propertyTemplate.languageSuppressed),
         null,
```

Comment:
we need to pass in this property of the template so we know how to set the default language correctly